Practice/index.html

```diff
@@ -4,68 +4,261 @@
+            </p>
+          </div>
+          <div
+            class="
+              col-lg-3 col-md-6 col-sm-12
+              d-flex
+              flex-column
+              align-items-center
+            "
+          >
+            <i class="far fa-heart fa-4x text-primary"></i>
+            <h3 class="py-3">Made with Love</h3>
+            <p>
+              Because nothing says hard work and talent like editing a few lines
+              of text.
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
 
+    <!-- bullshit -->
+    <section id="bullshit">
+      <div class="container-fluid">
+        <div class="row">
+          <img
+            class="col-xl-3 col-lg-6 col-md-12 gx-0"
+            src="./assets/image/1.jpg"
+          />
+          <img
+            class="col-xl-3 col-lg-6 col-md-12 gx-0"
+            src="./assets/image/2.jpg"
+          />
+          <img
+            class="col-xl-3 col-lg-6 col-md-12 gx-0"
+            src="./assets/image/3.jpg"
+          />
+          <img
+            class="col-xl-3 col-lg-6 col-md-12 gx-0"
+            src="./assets/image/4.jpg"
+          />
+        </div>
+      </div>
+    </section>
+
+    <!-- some section -->
+    <div
+      class="
+        container-fluid
+        d-inline-flex
+        flex-column
+        align-items-center
+        text-white
+        bg-dark
+        p-3
+      "
+    >
+      <h2 class="p-4">Seriously, It's This Fucking Template !</h2>
+      <a class="p-2 my-4 btn rounded-pill bg-white" href="#top"
+        >That's amazing</a
+      >
+    </div>
+    <!-- call me -->
+
+    <section id="callMe">
+      <div class="container-fluid py-5">
+        <div class="row d-flex justify-content-center">
+          <div
+            class="col-8 d-flex flex-column align-items-center text-center py-5"
+          >
+            <h2>My Contact Information</h2>
+            <hr class="py-1 px-5" />
+            <p>
+              Here's my phone number and email address, because hey that's part
+              of the template and why would you not include that on your
+              webpage? You'd be a fool not to! I can't wait for all the job
+              offers to come pouring in from all the companies wanting my insane
+              website design skills.<br />Honestly this template does look
+              really nice, though.
+            </p>
+          </div>
+        </div>
+        <div class="row d-flex justify-content-center text-center">
+          <div class="col-sm-6 col-md-4">
+            <i class="fas fa-globe-africa fa-4x text-primary"></i>
+            <a href="tel:+1-608-317-1888">+1-608-317-1888</a>
+          </div>
+          <div class="col-sm-6 col-md-4">
+            <i class="fas fa-globe-africa fa-4x text-primary"></i>
+            <a href="mailto:holy@shit.com">holy@shit.com</a>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- Bootstrap Bundle JS CDN-->
     <script

```

Comment:
I can see that you understand how to use bootstrap and javascript quite well. Well done 
Please come back to finish off the some of the features just to practice proper js and css